kafka/config.go

```diff
@@ -237,6 +237,32 @@ func (m ConfigMap) extract(key string, defval ConfigValue) (ConfigValue, error)
 	return v, nil
 }
 
+// extractLogConfig extracts generic go.logs.* configuration properties.
+func (m ConfigMap) extractLogConfig() (logsChanEnable bool, logsChan chan LogEvent, err error) {
```

Comment:
I would call this `createLogChannel` as it is not only extracting the log config, but also has the side effect of creating the channel itself. Side effects should always be explicit, I think.

Reply:
It doesn't, the logs channel is created later from setupLogQueue()